lib/bandit/http2/adapter.ex

```diff
@@ -147,7 +149,7 @@ defmodule Bandit.HTTP2.Adapter do
     headers = Bandit.Headers.add_content_length(headers, body_bytes, status)
 
     adapter =
-      if body_bytes == 0 do
+      if body_bytes == 0 || !send_resp_body?(adapter, status) do
```

Comment:
They're both booleans

```suggestion
      if body_bytes == 0 or !send_resp_body?(adapter, status) do
```

Reply:
Lots of prior art for || usage in this case; I generally try to avoid using `or` except when I *have* to